docs/docs-beta/docs/guides/external-systems/non-python.md

```diff
@@ -1,4 +1,66 @@
 ---
 title: "Running non-Python languages"
 sidebar_position: 60
----
\ No newline at end of file
+---
+
+Dagster is written in Python, but that doesn't mean it's limited to running Python to materialize assets. With Pipes, you can run code in other languages and send information back forth with Dagster. This guide covers how to run JavaScript with Dagster using Pipes, however, the same principle will apply to other languages.
+
+<details>
+<summary>Prerequisites</summary>
+
+- Familiarity with [Assets](/concepts/assets)
+- A basic understanding of JavaScript and Node.js
+</details>
+
+## Create a script using Tensorflow in JavaScript
```

Comment:
nit (take or leave it): should this guide follow the guide-with-steps so it's clearly step-by-step?